### DISCUSSION TITLE
Welcome to CPM.cmake Discussions!

### DISCUSSION THREAD

**Discussion by TheLartians** [cpm-cmake/CPM.cmake#184] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheLartians** [comment]
Hey everyone 👋, 

I'm Lars and have been coding C++ for over 10 years, both for my [studies](https://thelartians.de/) and [professionally](https://www.linkedin.com/in/lars-melchior/). After becoming more and more frustrated with using monorepos or submodules for dependency management and finding the "typical" external C++ dependency managers lacking in many ways at the time (e.g. Vcpkg had no proper version locking and Conan didn't properly support git), I decided to write this script to allow us a simple way to manage dependencies using only CMake. Since then it became my primary package manager for C++, as it fulfils (almost) all my needs and I see many advantages in combining the build system with the dependency manager. I also hope this allows us to more easily collaborate and share open-source projects with dependencies. I hope you find it as useful as I do!

Thanks to @iboB for suggesting to open up the discussions!

**Reply by ghost** [reply]
The newest update (0.28.0) drastically improved my visual studio configuration time! 

I can definitely see using CPM for our company in the future 👍🏽

**Reply by TheLartians** [reply]
That's awesome, thanks for the feedback! :)
